.mergify.yml

```diff
@@ -2,7 +2,7 @@ pull_request_rules:
   - name: automatically merge Scala Steward PRs on CI success
     conditions:
       - author=scala-steward
-      - status-success=Travis CI - Pull Request
+      - status-success=Build and Test
```

Comment:
Looks like this is problematic on cross-building, see e.g. [cats-effect's config](https://github.com/typelevel/cats-effect/blob/6e301c236fbf66c4d9739b2ae7fefe1a1c7bf696/.mergify.yml) - could you adapt the same pattern here to make sure we have all the Scala versions passing before Mergify tries to merge?

There's a feature request to get this in sbt-github-actions but it'll probably take a while: https://github.com/djspiewak/sbt-github-actions/issues/46

Reply:
oh and I'm aware this could also be a problem in the other repositories I provided as inspiration - fixing that in these would also be very welcome, if you want to take a stab at that. :)

Reply:
OK I'll take a look.